cmd/kanctl/main.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 Kasten Inc.
```

Comment:
@joshiomkarj - this should be `Copyright 2019 The Kanister Authors.`

Reply:
@vkamra Okay will fix this.

Reply:
@vkamra This is fixed now.